### DISCUSSION TITLE
Remove SDB support?

### DISCUSSION THREAD

**Discussion by JRMeyer** [coqui-ai/STT#2151] [General]
Looking through the augmentation docs today, I saw reference to `.sdb` files and thought we had already removed support for the `Sample Database` file format

I now see that's not the case, and we still have code for use of `sdb` files. However, as far as I know no one is using this custom file format, and given that we are moving to more modern data input methods (e.g. streaming of webdatasets), the `sdb` file format feels like a legacy approach we could remove to keep the code and docs leaner.

thoughts?

**Comment by reuben** [comment]
We'll take silence as evidence of no use :)
